src/edfi-paging-test/edfi_paging_test/api/paginated_result.py

```diff
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from typing import Any, Dict, List, Optional
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from edfi_paging_test.api.request_client import RequestClient
```

Comment:
If I have seen this before, then I have forgotten about it. Why not just import it?

Reply:
Causes circular import